hayeon/programmers-1/두-개-뽑아서-더하기.js

```diff
@@ -0,0 +1,41 @@
+function solution(numbers) {
+  const newNum = [];
+  for (let i = 0; i < numbers.length; i++) {
+    for (let j = i + 1; j < numbers.length; j++) {
+      newNum.push(numbers[i] + numbers[j]);
+    }
+  }
+  const arrNum = [...new Set(newNum)];
+  let result = arrNum.sort((a, b) => a - b);
+  return result;
+}
```

Comment:
```suggestion
function solution(numbers) {
  const newNums = []; // 배열이니까 Nums가 좋을 것 같습니다 ㅎㅎ
  for (let i = 0; i < numbers.length; i++) {
    for (let j = i + 1; j < numbers.length; j++) {
      newNum.push(numbers[i] + numbers[j]);
    }
  }
  const deduplicatedArr = [...new Set(newNum)]; // 중복 제거를 의미하는 deduplication 단어 사용
  const result = arrNum.sort((a, b) => a - b); // 재할당을 하지 않기때문에 const로 선언이 좋을 것 같습니다 ㅎㅎ
  return result;
}
```
- 아래 코드처럼 result 변수 없이 곧바로 리턴해도 됩니다.
```js
return arrNum.sort((a, b) => a - b);
```

Reply:
헉 ㅜㅜ 자세한 리뷰 너무 감사드립니다...진짜!!!! 무조건 복습하고 한번 더 풀어보겠습니닷!!